flink-runtime/src/main/java/org/apache/flink/runtime/concurrent/Executors.java

```diff
@@ -59,6 +71,219 @@ public void execute(@Nonnull Runnable command) {
 		}
 	}
 
+	public static ExecutorService directExecutorService() {
```

Comment:
I think we do not need the above `directExecutor()` method and `DirectExecutor` class. `directExecutorService()` can be used instead of `directExecutor()`.